### DISCUSSION TITLE
Problem with links path

### DISCUSSION THREAD

**Discussion by olicendr** [clientIO/joint#1881] [Q&A]
Hello!
I have a problew witk links path.
I am expecting this link result
![chrome_Rbjb4ORHGw](https://user-images.githubusercontent.com/90215998/200784139-fe4b1fa2-bfb1-4f32-8ad4-bfc69fa367f1.png)
but instead i am having this link result 
![chrome_7379lav2o6](https://user-images.githubusercontent.com/90215998/200784327-679b7542-fae2-46ee-b454-1d54fed8bc2e.png)
I guess the problem is that link counts all of this block height and renders considering this height
![chrome_UsI0MoD1dD](https://user-images.githubusercontent.com/90215998/200784777-d30008f0-3d89-4468-bbbf-71dd6d1e7c7d.gif)
and this block behaves correctly, svg rendered as expected
is there any way to correct this links behavior?

**Comment by kumilingus** [comment]
Could you please create a JSFiddle that reproduces the issue? It's not clear how the shapes are defined.
